### PR TITLE
ci : Fix broken links errors

### PR DESCRIPTION
**Pull request by TaKO8Ki**
## Issue number and link

Fixes deadlinks errors

## Describe your changes

- Bump up pest version
- Remove log files for deadlinks
- Run `cargo doc` in advance

## Checklist before requesting a review

- [x] I follow the [Semantic Pull Requests](https://github.com/zeke/semantic-pull-requests) rules (bugfix/feature)
- [ ] I specified links to related issues (must: bugfix, want: feature)
- [ ] I have performed a self-review of my code (bugfix/feature)
- [ ] I have added thorough tests (bugfix/feature)
- [ ] I have edited `## [Unreleased]` section in `CHANGELOG.md` following [keep a changelog](https://keepachangelog.com/en/1.0.0/) syntax (bugfix/feature)
- [ ] I {made/will make} a related pull request for [documentation repo](https://github.com/SpringQL/SpringQL.github.io) (feature)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -300,7 +300,7 @@ All other sections are for end-users.
 [Unreleased]: https://github.com/SpringQL/SpringQL/compare/v0.17.1...HEAD
 [Released]: https://github.com/SpringQL/SpringQL/releases
 [v0.18.1]: https://github.com/SpringQL/SpringQL/compare/v0.18.0...v0.18.1
-[v0.18.0]: https://github.com/SpringQL/SpringQL/compare/v0.17.2...v0.18.0
+[v0.18.0]: https://github.com/SpringQL/SpringQL/compare/v0.17.1...v0.18.0
```
Comment (laysakura) [inline CHANGELOG.md:303]: sorry, thanks!

File: Makefile.toml [hunk 2]
```diff
@@ -85,19 +85,11 @@ cargo d
...
-cargo deadlinks --check-http --ignore-fragments | tee target/cargo-deadlinks.out
-set -e
-# sort uniq for `cargo-deadlinks` output
-cat target/cargo-deadlinks.out | sort | uniq > target/cargo-deadlinks.sort
-# check diff for `deadlinks.expected`
-cat deadlinks.expected | sort | uniq > target/cargo-deadlinks.expected.sort
-diff -u target/cargo-deadlinks.expected.sort target/cargo-deadlinks.sort
-
+cargo doc
+for crate in $( ls target/doc | grep springql ); do
```
Comment (laysakura) [inline Makefile.toml:89]: I heard from @TaKO8Ki `cargo doc --no-deps` may be buggy so we might better to use just `cargo doc`.
Pretty good.

Could you consider to filter the target directories from crate names? (our crates might not include `springql` substring)

Comment (TaKO8Ki) [reply Makefile.toml]: You're right. I fixed it to get a list of crate names using `cargo-metadata`.

**Review by laysakura — APPROVED** [review]
Great work! Thank you!

**Pull request merged**